### PR TITLE
added documentation for oracle data integration

### PR DESCRIPTION
**Pull request by curranConcepts**
## Description

added documentation on the oracle data integration handler.

Fixes #(issue)
issue: #4885 

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] 📄 This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation, or created issues to update them.
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] I have shared a short loom video or screenshots demonstrating any new functionality.

### CODE REVIEW

**Comment by martyna-mindsdb** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

File: docs/data-integrations/oracle.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+---
+
+##Oracle handler
+
+This is the implementation of the Oracle handler for MindsDB.
+
+##Implementation
+This handler is implemented using the oracledb library. Please install it before using this handler:
+
+pip install oracledb
```
Comment (martyna-mindsdb) [inline docs/data-integrations/oracle.mdx:13]: Please enclose this command in a code block as follows:

> \```bash
> pip install oracledb
> \```

File: docs/data-integrations/oracle.mdx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+* `host`: the host name or IP address 
+* `port`: the TCP/IP port number
+* `sid`: the unique identifier of the database instance
+* `user`: the database username 
+* `password`: the password to authenticate the user 
+
+##Usage
+The following syntax can be used, in order to connect to the Oracle DB:
+
+CREATE DATABASE oracle_datasource         --- display name for the database
```
Comment (martyna-mindsdb) [inline docs/data-integrations/oracle.mdx:25]: Please enclose this command in a code block as follows:

> \```sql
> CREATE DATABASE oracle_datasource         --- display name for the database
> WITH ENGINE = 'oracle',                   --- name of the MindsDB handler
> PARAMETERS = {
>   "host": " ",                            --- host name or IP address
>   "port": ,                               --- port used to make TCP/IP connection
>   "sid": " ",                             --- unique identifier of the database instance
>   "user": " ",                            --- database user
>   "password": " "                         --- database password
> };
> \```

File: docs/data-integrations/oracle.mdx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  "sid": " ",                             --- unique identifier of the database instance
+  "user": " ",                            --- database user
+  "password": " "                         --- database password
+};
+
+The connection accepts either sid or service_name arguments to target to the right DB instance.
+
+Now, you can use this established connection to query your database as follows:
+
+SELECT * FROM oracle_db.my_table
```
Comment (martyna-mindsdb) [inline docs/data-integrations/oracle.mdx:39]: Please enclose this command in a code block as follows:

> \```sql
> SELECT * FROM oracle_db.my_table;
> \```

**Comment by martyna-mindsdb** [comment]
@curranConcepts 
Before we can merge your PR, please sign our CLA following the instructions above.

Please implement the changes requested in the review comments above. Thanks.

**Comment by curranConcepts** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**